src/openrct2/ride/Ride.cpp

```diff
@@ -8309,7 +8969,8 @@ void fix_ride_entrance_and_exit_locations()
             {
                 tileElement = map_get_ride_exit_element_at(exitLoc.x * 32, exitLoc.y * 32, entranceExitHeight, false);
 
-                if (tileElement == NULL || tileElement->properties.entrance.ride_index != rideIndex || tile_element_get_station(tileElement) != stationIndex)
+                if (tileElement == NULL || tileElement->properties.entrance.ride_index != rideIndex ||
```

Comment:
Shouldn't this be `== nullptr`?
Also, I'd prefer if this if statement was either vertical or horizontal.